Report/create_reports.R

```diff
@@ -60,8 +61,6 @@ state_geos <- locations %>%
   filter(nchar(.data$geo_value) == 2) %>%
   pull(.data$geo_value)
 signals <- c(
-  "confirmed_incidence_num",
-  "deaths_incidence_num",
   "confirmed_admissions_covid_1d"
```

Comment:
@dsweber2 All of these signals are processed. The looping mechanism exists in `evalcast`, we just need to pass a vector of signal names. I dropped the first two so they aren't processed.